src/main/common/filter.h

```diff
@@ -40,6 +42,14 @@ typedef struct biquadFilter_s {
     float x1, x2, y1, y2;
 } biquadFilter_t;
 
+typedef struct laggedMovingAverage_s {
+    uint16_t movingWindowIndex;
+    uint16_t windowSize;
+    float weight;
+    float movingSum;
+    float buf[MAX_LMA_WINDOW_SIZE];
```

Comment:
Been thinking about this a bit.  It might make it more flexible to declare this as a pointer and statically declare the buffer in gyro.c. This way, the buffer size is not determined by the struct and can be configured depending on the use case (use cases aren't limited to gyro filtering).  Might also make sense to move MAX_LMA_WINDOW_SIZE to gyro.c as well.  The init function should parameterize the pointer so it can be configured as required by the implementing module.